joomla/joomla-cms
# [4.0][RTL][Atum] Add RTL Support

Pull Request for Issue # .

### Summary of Changes
Broad RTL styling for the Atum backend template.

- General RTL changes
- Moved sidebar to right
- Flipped main navigation
- Flipped status bar
- Flipped quick icons
- Removed directional bootstrap classes
- Removed floats from table
- Refactored dashboard modules to allow RTL

### Testing Instructions
Apply patch and log in to admin with RTL language (eg. Arabic).

**Edit**: Can also be tested without having to install an RTL language. See.. https://github.com/joomla/joomla-cms/pull/16593#issuecomment-307340596

### Expected result
![rtl1](https://user-images.githubusercontent.com/2803503/26968995-0e790b40-4cfc-11e7-844b-ff30bacc0c8c.png)

![rtl2](https://user-images.githubusercontent.com/2803503/26969003-13ad2e5c-4cfc-11e7-8b3d-f43842a0db25.png)

![rtl3](https://user-images.githubusercontent.com/2803503/26969052-44eb0f98-4cfc-11e7-9424-8d66ea8a269d.png)

### Documentation Changes Required
Screenshots
